.storybook/main.js

```diff
@@ -39,4 +39,7 @@ module.exports = {
       },
     },
   ],
+  core: {
+    builder: 'webpack5',
+  },
```

Comment:
`@reach/auto-id` uses the `??` operator which was causing build errors. I was deciding between updating the storybook webpack config and just using the webpack5 builder. I think using the webpack5 builder is more simple.

Reply:
Think with webpack5 builder we can also remove the `NODE_OPTIONS=--openssl-legacy-provider` in `package.json`